examples/meta/src/binary/mkl_binclass.sg.in

```diff
@@ -36,7 +36,7 @@ mkl.train()
 Features feats_pred = create_features("CombinedFeatures")
```

Comment:
minor: could you change the filename to be consistent with the style we use. In this  case simply "multiple_kernel_learning.sg" (folder is already called binarry)

Reply:
it seems like we already have a meta example called "multiple_kernel_learning.sg".

Reply:
in the "binary" folder? If it is in another one, that is fine as the meta examples are named as foldername-example_name in cmake

Reply:
ah no it is in the same folder...
Could you then check whether they are doing the same thing (maybe we just forgot to delete it). If so, no need to add another one. If there is something that is missing from the old one, feel free to merge the two (if it makes sense).

Reply:
@LiuYuHui  have you checked if there is anything that needs to be added? If this doesn't add anything can just delete the legacy example and no need to change the meta example

Reply:
not completely same, but very similar

Reply:
if nothing new is tested then just keep the old meta example and delete the legacy example as well as the one you wrote